web/templates/pages/project_detail.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+
+{% block content%}
+	<div class="row">
+		<h3><a href="{% url 'web.views.project_detail' project_id=project.project_id %}">{{project.project_name}}</a></h3>
+	</div>
+	<div class="row">
+		<div class="col-md-4 col-xs-6">
+			<p><strong>project members: </strong></p>
+			<p><img src="http://placekitten.com/550/550" alt="" class="img-responsive"></p>
+		</div>
+		<div class="col-md-8 col-xs-6">
+			<p><strong>Project launch date: </strong>{{project.date_created|date:"d.m.Y"}}</p>
+			<p><strong>Current status: </strong>{{project.get_status_id_display}}</p>
+			{% if project.project_deadline != None %}
```

Comment:
It would be better to ask:

```
{% if project.project_deadline %}
```
bcs if it is None than this will return False anyway.